WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPublishSettingsFragment.kt

```diff
@@ -38,159 +19,6 @@ class EditPostPublishSettingsFragment : Fragment() {
                 .get(EditPostPublishSettingsViewModel::class.java)
     }
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```

Comment:
Could you please create a separate PR where you just create the parent fragment and move the logic away from this fragment? It's very difficult to check what you just moved vs changed. Thank you ! ;)

Reply:
Yes, I did that here :) https://github.com/wordpress-mobile/WordPress-Android/pull/11728